ServerRules.md

```diff
@@ -0,0 +1,11 @@
+## Server Rules
+
+1. Don't do anything you wouldn't do in Oso Negro
+2. Read and follow the Code of Conduct (https://nelson.social/about/more#code-of-conduct)
+3. No racist, sexist, homophobic, transphobic, ableist, or otherwise discriminatory or hateful jokes or language
+4. Don't post or threaten to post other people’s personally identifying information (i.e., doxing)
+5. Don't share intentionally false or misleading information
+6. Don't spam or engage in excessive commercial posting (business accounts are allowed, though)
+7. Don't do anything illegal in British Columbia or Canada
```

Comment:
Do we really have to mention this?

Reply:
🤷‍♀️ in theory as long as we reference the CoC and the incoming ToS, we don't need anything else in this list at all. But most people will probably not go and read those documents. I think our goal here is to give them the tl;dr.

Are you thinking we don't need this because it's obvious?

Reply:
"Please don't do anything illegal" comes across as a bit infantilizing to me.